src/main/resources/reference.conf

```diff
@@ -0,0 +1,16 @@
+com.crobox.clickhouse.client {
+  //  can be single-host, cluster-aware, balancing-hosts
+  connection {
+    type = "single-host"
+    host = "localhost"
+    port = 8123
+    //    this is only used for cluster aware connection
+    cluster = "cluster"
+    health-check-interval = 5
+  }
+
+  //  If the client should use http compression in the communication with clickhouse. More info: http://clickhouse-docs.readthedocs.io/en/latest/interfaces/http_interface.html
+  httpCompression = false
```

Comment:
Should we use normal-dash instead of camelCase for consistency in configuration?